desktop/libs/indexer/src/data/oozie_workspace/clean_to_match_schema.conf

```diff
@@ -12,7 +12,8 @@
 # distributed under the License is distributed on an "AS IS" BASIS,
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
-# limitations under the License.from nose.tools import assert_equal
```

Comment:
Somehow the import statement had crept into the license text. Good side effect of the cleaning.

Reply:
Yes, agreed!